components/Layout/navigation.js

```diff
@@ -1,16 +1,46 @@
-import { StyledFooter, StyledLink } from "@/lib/styled-components";
-export default function Navigation({ className, session }) {
+import { StyledFooter, StyledNavLink } from "@/lib/styled-components";
+import { useRouter } from "next/router";
+import Image from "next/image";
+import addRoastIcon from "@/public/addRoastIcon.svg";
+import homeIcon from "@/public/homeIcon.svg";
+import searchIcon from "@/public/searchIcon.svg";
+import profileIcon from "@/public/profileIcon.svg";
+
+export default function Navigation({ className }) {
+  const router = useRouter();
   return (
     <StyledFooter className={className}>
-      <StyledLink href={"/"}>Startseite</StyledLink>
-      <StyledLink href={"/search"}>Suchen</StyledLink>
-      <StyledLink href={"/addroast"}>
-        Kaffee <br />
-        hinzufügen
-      </StyledLink>
-      <StyledLink href={"/login"}>
-        {session ? "Favoriten" : "Profil"}
-      </StyledLink>
+      <StyledNavLink $active={router.route === "/" ? true : false} href={"/"}>
```

Comment:
You can use the shorthand if you like.
```suggestion
      <StyledNavLink $active={router.route === "/"} href={"/"}>
```